server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/UpdateMobileBuildHelper.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Arcadia.Assistant.AppCenterBuilds.Contracts.AppCenter;
+using Arcadia.Assistant.MobileBuild.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Arcadia.Assistant.AppCenterBuilds
+{
+    internal class UpdateMobileBuildHelper
```

Comment:
what's the reason for this class to be internal? Please make it public so if (I don't believe it but anyway) we want to test - we're able to.

Reply:
Done